### ISSUE TITLE
Input not included in command line due to inputBinding field position

### ISSUE THREAD

**Issue by bogdang989** [rabix/bunny#128]
Tool has an input defined like this:

```yaml
inputs:
  - id: source_sqlite
    format: "edam:format_3621"
    type:
      type: array
      items: File
      inputBinding:
        prefix: "--source_sqlite"
```

This input is never included in the command line. Adding another inputBinding field one level up (with prefix: "" for example) solves this, but I think this original input definition is also valid. Is that correct?

**Comment by bogdang989** [comment]
Here is an example for this 
[example.zip](https://github.com/rabix/bunny/files/771677/example.zip)

**Comment by milos-ljubinkovic** [comment]
Should be fixed on v1.0 with pull request https://github.com/rabix/bunny/pull/194
